### PR TITLE
Update link that was failing on CI

### PR DESCRIPTION
**Pull request by willingc**
----
📚 Documentation preview 📚: https://python-packaging-user-guide--1507.org.readthedocs.build/en/1507/

### CODE REVIEW

File: source/tutorials/installing-packages.rst [hunk 1]
```diff
@@ -457,7 +457,7 @@ directory to your :envvar:`PATH`:
   ``PATH`` changes to take effect.
 
 .. _user base: https://docs.python.org/3/library/site.html#site.USER_BASE
-.. _modifying ~/.profile: https://stackoverflow.com/a/14638025
+.. _modifying ~/.profile: https://stackoverflow.com/questions/14637979/how-to-permanently-set-path-on-linux-unix/14638025#14638025
```
Comment (willingc) [inline source/tutorials/installing-packages.rst:460]:
```suggestion
```

**Comment by webknjaz** [comment]
Why do you think it doesn't?

**Comment by webknjaz** [comment]
@dukecat0 @jwodder @theacodes since you were the ones to touch/add that link, do you have any opinion on keeping or dropping it?

**Comment by dukecat0** [comment]
I personally prefer keeping it as a quick reference for beginners, who are the main audience of this tutorial.

**Comment by webknjaz** [comment]
Yeah, that's my concern as well. I prefer keeping the reference and ignoring the link instead.

**Comment by willingc** [comment]
Sounds good. I'll add back the link.

**Comment by willingc** [comment]
Close and replace with #1510 to ignore stack overflow for linkchecks

**Pull request closed without merging**